fdb-record-layer-icu/fdb-record-layer-icu.gradle

```diff
@@ -22,22 +22,24 @@ apply from: rootProject.file('gradle/strict.gradle')
 
 def coreProject = ":${ext.coreProjectName}"
 dependencies {
-    compile project(coreProject)
-    compile "com.ibm.icu:icu4j:${icuVersion}"
-    compile "com.google.protobuf:protobuf-java:${protobufVersion}"
-    compile "org.slf4j:slf4j-api:${slf4jVersion}"
+    implementation project(coreProject)
```

Comment:
I feel like this should probably be `api` instead of `implementation`. This subproject provides an implementation of an interface defined in `fdb-record-layer-core`, so I think it needs to say that it takes on the core project as an `api` dependency, but I'm not actually certain.

Reply:
I am fine with that.

Reply:
changed to `api`